capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.servlets;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.cloud.firestore.QuerySnapshot;
+import com.google.cloud.firestore.SetOptions;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.cloud.FirestoreClient;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+
+
+public class Utility {
```

Comment:
Consider making specific utilities for different purposes (e.g. `DatabaseUtility` holding database-related methods) in the future if this class grows.

Reply:
That's a good idea, and will definitely think about breaking this class apart if it gets unwieldy.